packages/java/parser-jvm-plugin-backbone/src/main/java/com/vaadin/hilla/parser/plugins/backbone/EndpointExposedPlugin.java

```diff
@@ -103,10 +103,8 @@ public NodeDependencies scan(@NonNull NodeDependencies nodeDependencies) {
     }
 
     private boolean alwaysExpose(ClassInfoModel classInfo) {
-        return classInfo
-                .is("com.vaadin.flow.spring.data.jpa.CrudRepositoryService")
-                || classInfo.is(
-                        "com.vaadin.flow.spring.data.jpa.ListRepositoryService");
+        return classInfo.is("com.vaadin.hilla.crud.CrudRepositoryService")
```

Comment:
Why? Aren't the Hilla classes normal `@EndpointExposed`?

Reply:
Correct. I went too fast there.